src/test/java/net/sf/jabref/importer/CustomImporterTest.java

```diff
@@ -0,0 +1,120 @@
+package net.sf.jabref.importer;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.importer.fileformat.BibTeXMLImporter;
+import net.sf.jabref.importer.fileformat.CopacImporter;
+import net.sf.jabref.importer.fileformat.OvidImporter;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class CustomImporterTest {
+
+    private CustomImporter importer;
+
+    @Before
+    public void setUp() {
+        Globals.prefs = JabRefPreferences.getInstance();
+
+        importer = new CustomImporter(new CopacImporter());
+    }
+
+    @Test
+    public void testGetName() {
+        assertEquals("Copac", importer.getName());
+    }
+
+    @Test
+    public void testGetCliId() {
+        assertEquals("cpc", importer.getClidId());
+    }
+
+    @Test
+    public void testGetClassName() {
+        assertEquals("net.sf.jabref.importer.fileformat.CopacImporter", importer.getClassName());
+    }
+
+    @Test
+    public void testGetBasePath() {
+        assertEquals("src/main/java/net/sf/jabref/importer/fileformat/CopacImporter.java", importer.getBasePath());
+    }
+
+    @Test
+    public void testGetInstance() throws Exception {
+        assertEquals(new CopacImporter(), importer.getInstance());
+    }
+
+    @Test
+    public void testGetFileFromBasePath() {
+        assertEquals(new File("src/main/java/net/sf/jabref/importer/fileformat/CopacImporter.java"), importer.getFileFromBasePath());
+    }
+
+    @Test
+    public void testGetBasePathUrl() throws Exception {
+        assertEquals(new File("src/main/java/net/sf/jabref/importer/fileformat/CopacImporter.java").toURI().toURL(), importer.getBasePathUrl());
+    }
+
+    @Test
+    public void testGetAsStringList() {
+        assertEquals("Copac", importer.getAsStringList().get(0));
+        assertEquals("cpc", importer.getAsStringList().get(1));
+        assertEquals("net.sf.jabref.importer.fileformat.CopacImporter", importer.getAsStringList().get(2));
+        assertEquals("src/main/java/net/sf/jabref/importer/fileformat/CopacImporter.java", importer.getAsStringList().get(3));
+    }
+
+    @Test
+    public void testEqualsTrue() {
+        boolean test = importer.equals(importer);
```

Comment:
Because otherwise codacy wants me to use assertEquals(). But I want to test equals() from CustomImporter. not the normal equals().